migrations/2021-08-28-150018_create_teams/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+CREATE TABLE teams (
+    id SERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    name TEXT NOT NULL,
+    avatar TEXT,
+    personal BOOLEAN NOT NULL DEFAULT TRUE
```

Comment:
This should be `DEFAULT FALSE` since a user will really only have one personal team, so it's better to special-case that as compared to normal team creation